src/test/java/htsjdk/tribble/AbstractFeatureReaderTest.java

```diff
@@ -228,4 +229,45 @@ public SeekableByteChannel truncate(long size) throws IOException {
         }
     }
 
+    @DataProvider
+    public Object[][] getVcfRedirects(){
+        return new Object[][]{
+          {"src/test/resources/htsjdk/tribble/AbstractFeatureReaderTest/vcf.redirect"},
+          {"src/test/resources/htsjdk/tribble/AbstractFeatureReaderTest/vcf.gz.redirect"}
+        };
+    }
+
+    @Test(dataProvider = "getVcfRedirects")
+    public void testRedirectCode(String vcfRedirect) throws IOException {
```

Comment:
Add an explanatory comment above this test explaining what it's testing, since the name doesn't make it obvious